src/views/manage/ManageForm.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Button, Heading, Row, Column, Typography } from '@datapunt/asc-ui';
+import { Formik } from 'formik';
+import ManageFormStyle, { Label, StyledColumn } from './ManageFormStyle';
+import FormFields, { initalValues } from './FormFields';
+
+const FormField = ({ name, label, Component, errors, ...otherProps }) => {
+  return (
+    <Label>
+      {label} {errors && `- ${errors}`}
+      <Component name={name} {...otherProps} />
+    </Label>
+  );
+};
+
+const ManageFormBase = ({
+  touched,
+  errors,
+  values,
+  handleChange,
+  handleBlur,
+  handleSubmit,
+  setFieldValue,
+}) => {
+  return (
+    <ManageFormStyle onSubmit={handleSubmit} action="" novalidate>
+      clg
+      <Row wrap>
+        <StyledColumn span={6} direction="vertical">
+          <Heading $as="h3" color="secondary">
+            Locatie
+          </Heading>
+          {FormFields.filter(({ column }) => column === 1).map(
```

Comment:
als je het formulier in 2 kolommen maakt dan kan je beter alle items onder elkaar in een (flex) container stoppen. dit maakt het nodeloos ingewikkeld

Reply:
It is not possible because the flow of the controls is vertical and not horizontal